pkgs/applications/misc/fitnesstrax/default.nix

```diff
@@ -0,0 +1,51 @@
+{ pkgs
+, fetchFromGitHub
+, stdenv
+, rustPlatform
+, atk
+, gnome2
+, cairo
+, glib
+, gdk_pixbuf
+, gtkd
+, gtk3
+, wrapGAppsHook
+}:
+rustPlatform.buildRustPackage rec {
+  name = "fitnesstrax";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "luminescent-dreams";
+    repo = "fitnesstrax";
+    rev = "fitnesstrax-0.1.0";
+    sha256 = "1k6zhnbs0ggx7q0ig2abcnzprsgrychlpvsh6d36dw6mr8zpfkp7";
+  };
+
+  buildInputs = [
+    atk
+    gnome2.pango
+    cairo
+    glib
+    gdk_pixbuf
+    gtkd
```

Comment:
where do you use dlang bindings?

Reply:
Sorry, I don't understand the question. What do you mean?

Reply:
gtdk is `D binding and OO wrapper for GTK` https://dlang.org/. Isn't this all in rust?

Reply:
Going to guess that was a leftover of some early flailing as I was trying to chase down all of the dependencies, and that I forgot about it.